devops/docker/ubuntu-14.04-amd64/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:trusty
```

Comment:
Is there a way to get this from MCR?

Reply:
Sadly it isn't present on MCR, they only go as far back as 18.04. There is a task to create mirrors for the containers not present in MCR as a follow-up.

Reply:
This is old Ubuntu 14, Robert.